blocks/templates/blocks.pot

```diff
@@ -1,495 +1,613 @@
 msgstr ""
 
+#: commands-online.txt:243 commands-offline.txt:243
 msgid "tilt"
 msgstr ""
 
+#: commands-online.txt:245 commands-offline.txt:245
 msgid "timer"
 msgstr ""
 
+#: commands-online.txt:247 commands-offline.txt:247
 msgid "touching %m.touching?"
 msgstr ""
 
+#: commands-online.txt:249 commands-offline.txt:249
 msgid "touching color %c?"
 msgstr ""
 
-msgid "turn @turnLeft %n degrees"
+#: commands-online.txt:251 commands-offline.txt:251
+msgid "turn %m.motor off"
 msgstr ""
 
-msgid "turn @turnRight %n degrees"
+#: commands-online.txt:253 commands-offline.txt:253
+msgid "turn %m.motor on"
 msgstr ""
 
-msgid "turn %m.motor off"
+#: commands-online.txt:255 commands-offline.txt:255
+msgid "turn %m.motor on for %n secs"
 msgstr ""
 
-msgid "turn %m.motor on"
+#: commands-online.txt:257 commands-offline.txt:257
+msgid "turn @turnLeft %n degrees"
 msgstr ""
 
-msgid "turn %m.motor on for %n secs"
+#: commands-online.txt:259 commands-offline.txt:259
+msgid "turn @turnRight %n degrees"
 msgstr ""
 
+#: commands-online.txt:261 commands-offline.txt:261
 msgid "turn video %m.videoState"
 msgstr ""
 
+#: commands-online.txt:263 commands-offline.txt:263
 msgid "username"
 msgstr ""
 
+#: commands-online.txt:265 commands-offline.txt:265
 msgid "video %m.videoMotionType on %m.stageOrThis"
 msgstr ""
 
+#: commands-online.txt:267 commands-offline.txt:267
 msgid "volume"
 msgstr ""
 
+#: commands-online.txt:269 commands-offline.txt:269
 msgid "wait %n secs"
 msgstr ""
 
+#: commands-online.txt:271 commands-offline.txt:271
 msgid "wait until %b"
 msgstr ""
 
+#: commands-online.txt:273 commands-offline.txt:273
 msgid "when %m.booleanSensor"
 msgstr ""
 
+#: commands-online.txt:275 commands-offline.txt:275
 msgid "when %m.key key pressed"
 msgstr ""
 
+#: commands-online.txt:277 commands-offline.txt:277
 msgid "when %m.sensor %m.lessMore %n"
 msgstr ""
 
+#: commands-online.txt:279 commands-offline.txt:279
 msgid "when %m.triggerSensor > %n"
 msgstr ""
 
+#: commands-online.txt:281 commands-offline.txt:281
 msgid "when @greenFlag clicked"
 msgstr ""
 
-# "backdrop" in Scratch 2.0 is similar to "background" in Scratch 1.4
+#: commands-online.txt:283 commands-offline.txt:283
 msgid "when backdrop switches to %m.backdrop"
 msgstr ""
 
+#: commands-online.txt:285 commands-offline.txt:285
 msgid "when distance %m.lessMore %n"
 msgstr ""
 
+#: commands-online.txt:287 commands-offline.txt:287
 msgid "when I receive %m.broadcast"
 msgstr ""
 
+#: commands-online.txt:289 commands-offline.txt:289
 msgid "when I start as a clone"
 msgstr ""
 
-msgid "when Stage clicked"
-msgstr ""
-
+#: commands-online.txt:291 commands-offline.txt:291
 msgid "when this sprite clicked"
 msgstr ""
 
+#: commands-online.txt:293 commands-offline.txt:293
 msgid "when tilt %m.eNe %n"
 msgstr ""
 
+#: commands-online.txt:295 commands-offline.txt:295
 msgid "x position"
 msgstr ""
 
+#: commands-online.txt:297 commands-offline.txt:297
 msgid "y position"
 msgstr ""
-
```

Comment:
Oops, I lied: these particular strings aren't in the Flash source since they're out of date. The up-to-date strings are in the Flash source, though, and they're in this file around [line 387](#diff-2c63731f1d120ceff7ae6ab82badcd13R387) and [line 519](#diff-2c63731f1d120ceff7ae6ab82badcd13R519). It doesn't look to me like any change is necessary after all.